src/span_context.js

```diff
@@ -142,8 +155,8 @@ export default class SpanContext {
         this._debugId = debugId;
     }
 
-    set samplingFinished(finished: boolean) {
-        this._samplingFinished = finished;
+    set samplingFinalized(finished: boolean) {
```

Comment:
I do not believe there is a valid use case to set samplingFinalized to false. Until we have such use case, I suggest `finalizeSampling(): void` method instead of a property setter.